src/main/resources/com/google/api/codegen/java/method_sample.snip

```diff
@@ -0,0 +1,157 @@
+@extends "common.snip"
```

Comment:
Maybe consider not pulling out method_sample.snip for java for now, due to my open PR which also changes the corresponding snippets.